### PR TITLE
This adds new trigger channels, as well as some small OH3 compatabili…

### PR DESCRIPTION
**Pull request by digitaldan**
…ty updates

Signed-off-by: Dan Cunningham <dan@digitaldan.com>

### CODE REVIEW

File: fluent/fluent.js [hunk 1]
```diff
@@ -207,7 +216,16 @@ const fluentExports = {
...
-    
+
+    /**
+     * Specifies an group member as the source of changes to trigger a rule.
+     * 
+     * @memberof fluent
+     * @param {String} groupName the name of the group
+     * @returns {ItemTriggerConfig} the trigger config
+     */
+    memberOf: s => new triggers.ItemTriggerConfig(s, true),
```
Comment (digitaldan) [inline fluent/fluent.js:227]: FYI memberOf is how this is used in our classic DLS rules engine as welll

File: fluent/trigger-conf.js [hunk 2]
```diff
@@ -130,7 +171,115 @@ class ItemTriggerConfig {
...
+    constructor() {
+        this._toOHTriggers = () => [triggers.SystemStartlevelTrigger(this.level)]
+        this.describe = (compact) => compact ? `system:${this.level}` : `system level "${this.level}"`
+    }
+    _complete() {
+        return typeof (this.level) !== 'undefined';
+    }
+
+    rulesLoaded() {
+        this.level = 40;
```
Comment (digitaldan) [inline fluent/trigger-conf.js:249]: I should probably just call `return this.startLevel(40)` for this and the other methods

**Comment by digitaldan** [comment]
@jpg0 if you could review that would be awesome!

**Review by jpg0 — APPROVED** [review]

**Comment by jpg0** [comment]
FYI you may want to remove the `utils.typeWithFallback` stuff - this was to explicitly support both OH 2 & 3 concurrently as things moved packages. It probably isn't necessary any longer as we are only targeting 3.

**Comment by digitaldan** [comment]
>FYI you may want to remove the utils.typeWithFallback

Agreed,  thanks for reviewing!

**Pull request merged**